pipeline/terraform/stack/provider.tf

```diff
@@ -3,15 +3,13 @@
 # See https://github.com/hashicorp/terraform/issues/25172#issuecomment-641284286
 terraform {
   required_providers {
-    aws = {
-      source = "hashicorp/aws"
-
-      configuration_aliases = [aws.catalogue]
-    }
-
     ec = {
       source  = "elastic/ec"
       version = "0.2.1"
     }
   }
 }
+
```

Comment:
I reverted this change cause it doesn't allow me to target in the new version, now sure why 🤷‍♀️